### ISSUE TITLE
Recording OpenArena game play is too dark on Linux

### ISSUE THREAD

**Issue by rilysh** [OpenArena/engine#88]
Hi,
I'm trying to record a OpenArena game play but no matter what I set, screen recording will always going to be dark, I mean too dark.

Even taking screenshot is also dark, for some reason. I tried to set gamma through xgamma (I'm using X11 and Xfce as DE), but no luck! Followed some old OpenArena forum posts, but I couldn't able to find anything that could work, especially the screen recording part.

- This is how it looks (screenshot)
![image](https://github.com/OpenArena/engine/assets/71683721/2b430ee0-756e-45ba-a282-b533f8e5f255)
- This is how it should look like 
![IMG_20240405_233732_919](https://github.com/OpenArena/engine/assets/71683721/49745770-7b12-4b92-8474-a7846b42c64e)

I'm using binaries downloaded from here: http://www.openarena.ws/download.php?list.61

Thanks.

**Comment by NeonKnightOA** [comment]
What's the output with a compiled engine?

I mean, if by compiling a product from this repo happens the same.

**Comment by rilysh** [comment]
I've tried upstream source just a few days ago, no differences, it's just the same.

**Comment by leilei-** [comment]
in the OpenArena engine (this git) there's r_alternateBrightness which should default to on, as that allows the overbrights to work again.

**Comment by leilei-** [comment]
closing this because this is not an issue with the current engine.

**Comment by rilysh** [comment]
This _is_ from the current tree, master branch. I'm not sure if you mean "current engine" by this: https://github.com/ioquake/ioq3

**Comment by The-Gig** [comment]
In the opening post Rilysh implied he's using OA 0.8.8 binaries (he linked 0.8.8 download page).
_I might be wrong_, but I think leilei is referring to the "master" branch of OA fork <https://github.com/OpenArena/engine>.
To test post-0.8.8 OA binaries, we have nightly builds for Windows. For Linux, I fear you may have to compile them by yourself...

**Comment by rilysh** [comment]
> In the opening post Rilysh implied he's using OA 0.8.8 binaries (he linked 0.8.8 download page). _I might be wrong_, but I think leilei is referring to "latest version" of the OA engine code.

I don't really understand what you mean "latest version" here. Git hosted source code should be the one that's the "latest version" and most updated. Download page at OpenArena website refers 0.8.8 the last "newest" release and the compiled binary (0.8.8) is approximately ~12 years old.

The only branch I found is active (last commit 2 months ago) is this one: https://github.com/OpenArena/engine/commits/leidev24/ which I haven't really tested yet. 

> To test _post-0.8.8 OA binaries_, we have nightly builds for Windows. For Linux, I fear you may have to compile them by yourself...

Can you provide me the source code link (and possibly the nightly builds URL, if available)? Isn't the source code hosted in the current repository at https://github.com/OpenArena/engine/ (this repo)? Sorry but I'm a bit confused with the statement.

**Comment by The-Gig** [comment]
With "latest version" I just meant up-to-date with "master" branch of this repository, indeed.
Did you only test the issue with 0.8.8 binaries or with up-to-date code, too? I'm not sure when that r_alternateBrightness 2 thing has been implemented, but probably after 0.8.8 release
.
The Windows engine nightly builds can be found here: <https://files.poulsander.com/~poul19/public_files/oa/dev088/engine_windows_nightly/>

**Comment by rilysh** [comment]
I tried both. The first one was 0.8.8 (from the website) and the second one was from this repository. Neither one addressed the problem.

I did try to increase the gamma, and it somewhat fixed the "recording being too dark" problem (even after the adjustment, it was dark, but wasn't too dark) but apparently gamma level needs to be adjusted to a very high level which makes the game play (from the players view) very bright.
